### ISSUE TITLE
Option to import private keys

### ISSUE THREAD

**Issue by leafcutterant** [darkwallet/darkwallet#40]
At the moment one can't import or export private keys. I don't know how good idea is this in a deterministic wallet, but I think it would be okay after a warning that the seed won't contain the imported keys.

**Comment by dabura667** [comment]
I agree with a swipe function for "importing" but warn them you can only send the whole balance to one of your deterministic addresses and then it deletes the address. etc.

As for EXPORT. I completely agree there must be some way to export privkeys.

Since these functions seem like they should be for more advanced users, requiring a dev console to perform them might be a good idea to stop people from doing unsafe things. (most people who don't understand security well will run away at the sight of a console where they have to type commands)

However, a swipe command could be a UI element. (not a command)

tl;dr:
IMPORT -1
EXPORT +1

**Comment by leafcutterant** [comment]
I think the user should be given the freedom to import a private key itself (not swiping), of course accompanied with warnings & educational info. There are cases when a user doesn't want to leave a trace on the blockchain by swiping, to conceal that the address is active/being used (anti-forensics!), or doesn't want to pay the tx fee for the swiping.

So, like Import dialogue -> Swipe address or import privkey itself, if latter selected, red warning dialogue appears that it's safer to swipe and only proceed if you're certain your key is not known by someone else and you know what you're doing, etc.

For the export part, I agree to make it console-only and of course require a password. Then we'll see if that trend should be continued into the main releases.

**Comment by bitbutter** [comment]
+1 for leafcutter's suggestion wrt a possible procedure when importing (not swiping/sweeping, which would also be a valuable option) private keys.

Private key import seems to me like an important feature for a 'complete' wallet.

**Comment by caedesvvv** [comment]
Export tool is already in, so changing the task name to reflect this.

**Comment by amingilani** [comment]
Import vs swipe?
![82733-why-not-both-meme-5lvd](https://cloud.githubusercontent.com/assets/1188552/4681904/20c2358a-5617-11e4-9359-abdfa15021a3.jpeg)

Also, I think Armory exports imported private keys when you make a backup. That would be nice.
